packages/apps/storybook/.env

```diff
@@ -1,2 +1,2 @@
 # Registered client which have access to scopes "imodels:read", "imodels:modify", "itwins:read" and "itwins:modify", redirectUrl should be http://localhost:6006/signin-oidc.html
-STORYBOOK_AUTH_CLIENT_ID=
+STORYBOOK_AUTH_CLIENT_ID=spa-9zFI59a0wdnl0DtNiHcICU9ie
```

Comment:
Should this be committed?

Reply:
Yeah, This PR is in draft for now. So need to fix things and it'll be available for review soon

Reply:
Use `.env.local` file instead of editing `.env`, and you should now probably delete this client and create a new one as this page is public. We might want to add a comment at the top of `.env` that explains this to prevent that type of commit.

Reply:
Use `.env.local` file instead of editing `.env`, and you should now probably delete this client and create a new one as this page is public.

Reply:
Removed the client and added comment on top of `.env`